DQM/L1TMonitor/python/L1TStage2uGTEmul_cfi.py

```diff
@@ -4,5 +4,9 @@
 l1tStage2uGtEmul = DQMEDAnalyzer('L1TStage2uGT',
     l1tStage2uGtSource = cms.InputTag("valGtStage2Digis"),    
     monitorDir = cms.untracked.string("L1TEMU/L1TStage2uGTEmul"),
-    verbose = cms.untracked.bool(False)
+    verbose = cms.untracked.bool(False),
+    firstBXInTrainAlgo = cms.untracked.string("L1_FirstCollisionInTrain"),
+    lastBXInTrainAlgo = cms.untracked.string("L1_LastCollisionInTrain"),
+    unprescaledAlgoShortList = cms.untracked.vstring("L1_SingleMu25","L1_SingleEG32","L1_DoubleMu_15_7","L1_SingleJet200","L1_HTT450er"),
```

Comment:
Please replace the list of algos with this one:
`unprescaledAlgoShortList = cms.untracked.vstring("L1_SingleMu22_BMTF”, “L1_SingleMu22_OMTF”, “L1_SingleMu22_EMTF”, “L1_SingleIsoEG30er1p5”, ”L1_SingleIsoEG32er2p5”, “L1_SingleEG40er2p5”, “L1_SingleEG60”, “L1_SingleTau120er2p1”, "L1_SingleJet180”, “L1_ETMHF130”, ”L1_HTT360er”, “L1_ETT2000”),`